### PR TITLE
Auto accept keys in docker-zypper calls

### PR DESCRIPTION
**Pull request by mloviska**
Again NVidia repo is causing troubles when container repos are being refreshed.

- Related ticket: https://progress.opensuse.org/issues/155665
- Verification run: http://kepler.suse.cz/tests/22662#step/image_docker/299

### CODE REVIEW

File: lib/containers/container_images.pm [hunk 1]
```diff
@@ -126,13 +126,14 @@ sub build_with_zypper_docker {
     zypper_call("in zypper-docker") if (script_run("which zypper-docker") != 0);
 
     my $log = "/tmp/zypper-docker-list-updates.log";
-    script_retry("zypper-docker list-updates $image >$log", timeout => 300, retry => 3, delay => 60);
+    my $zypp_dock = 'zypper-docker --gpg-auto-import-keys';
```
Comment (asmorodskyi) [inline lib/containers/container_images.pm:129]: Are you sure it is really necessary to include this key into every zypper call ? AFAIK if very first call will have it keys will be imported and all further calls won't ask any questions about keys anyway

Comment (ricardobranco777) [reply lib/containers/container_images.pm:129]: I'd rather use `zypper_docker` as variable name.

**Review by ricardobranco777 — APPROVED** [review]
Small suggestion. LGTM

File: lib/containers/container_images.pm (hunk 1 above)
Comment (mloviska) [reply lib/containers/container_images.pm:129]: That for sure works well when using zypper in a host system. Not sure whether docker or the container will save the state somehow. Maybe yes, but I find it as a safe action to include it in all calls

**Review by grisu48 — APPROVED** [review]
LGTM!

**Pull request merged**